fyle_accounting_mappings/urls.py

```diff
@@ -16,13 +16,14 @@
 from django.urls import path
 
 from .views import MappingSettingsView, MappingsView, EmployeeMappingsView, CategoryMappingsView,\
-    SearchDestinationAttributesView, MappingStatsView
+    SearchDestinationAttributesView, MappingStatsView, ExpenseAttributesMappingView
 
 urlpatterns = [
     path('settings/', MappingSettingsView.as_view()),
     path('employee/', EmployeeMappingsView.as_view()),
     path('category/', CategoryMappingsView.as_view()),
     path('destination_attributes/search/', SearchDestinationAttributesView.as_view()),
     path('stats/', MappingStatsView.as_view()),
-    path('', MappingsView.as_view())
+    path('', MappingsView.as_view()),
+    path('expense_attributes/mappings/', ExpenseAttributesMappingView.as_view())
```

Comment:
URL needs to change, 
this one will be like `mappings/expense_attributes/mappings/`

Reply:
maybe we can just make do with `mappings/expense_attributes/`